dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/importer/EventManager.java

```diff
@@ -248,6 +248,7 @@ public ImportSummaries updateEvents( final List<Event> events, final WorkContext
             catch ( Exception e )
             {
                 handleFailure( workContext, importSummaries, events, IMPORT_ERROR_STRING, UPDATE );
```

Comment:
I just checked that `handleFailure` will add an error in the import summary.
Its called 3 times after catching an exception like this.

Why did you decide logging the exception? If it is valuable, should we do this in all 3 cases and thus move this into `handleFailure`?

Reply:
I added the log because in the summary we collect just a general error, but we are missing important information for us in regard to why did it fail. You are right we could add the same log to the `addEvents` method, to the `delete` method I think it's not really useful as it doesn't do as many things as the other two, would you agree?

Reply:
After a closer examination, the `addEvents` method is already catching a general Exception, so I don't think a log is useful there either, I'll leave it in the `updateEvents` method only, for now.